tests/e2e/config/wdio.android.local.conf.js

```diff
@@ -0,0 +1,25 @@
+const {join} = require('path');
+const config = require('./wdio.shared.conf').config;
+
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
+  {
+    // The defaults you need to have in your config
+    automationName: 'UiAutomator2',
+    deviceName: 'Nexus_5_7.1.1',
```

Comment:
surprised we're testing on a device this old - Nexus 5 doesn't get updates any more

Reply:
Updated this to a Pixel One with 8.1 on it